homeassistant/components/vicare/climate.py

```diff
@@ -146,16 +146,16 @@ class ViCareClimate(ClimateEntity):
     _attr_max_temp = VICARE_TEMP_HEATING_MAX
     _attr_target_temperature_step = PRECISION_WHOLE
     _attr_preset_modes = list(HA_TO_VICARE_PRESET_HEATING)
+    _attributes: dict[str, Any] = {}
+    _current_action: bool | None = None
 
     def __init__(self, name, api, circuit, device_config):
         """Initialize the climate device."""
         self._attr_name = name
         self._api = api
         self._circuit = circuit
-        self._attributes = {}
         self._current_mode = None
         self._current_program = None
```

Comment:
While we're at it, can we also maybe type these?

Reply:
No idea what the api gives here, maybe `str`?

Reply:
If you don't know it, then this is okay